codecov.yml

```diff
@@ -0,0 +1,10 @@
+coverage:
+  range: 85..100
+  status:
+    project:
+      default:
+        threshold: 1%
+    patch:
+      default:
+        threshold: 1%
```

Comment:
Let’s keep indeed `target: auto` by now, but we may need to relax `patch` target vs `range` for the global project in the long term, being more lenient on the patches themselves.

I suggest we revisit those behaviours after some PR.